### ISSUE TITLE
module 'quimb.tensor' has no attribute 'ham_heis'

### ISSUE THREAD

**Issue by saraphys** [jcmgray/quimb#99]
Hello.
I just successfully installed qumib but getting the following error after running the example codes mentioned in quimb:

Code:
import quimb as qu
import quimb.tensor as qtn
import numpy as np
L = 44
zeros = '0' * ((L - 2) // 3)
binary = zeros + '1' + zeros + '1' + zeros
print('psi0:', f"|{binary}>")
psi0 = qtn.MPS_computational_state(binary)
psi0.show()  # prints ascii representation of state
 H = qtn.ham1d_heis(L)

Error:
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'quimb.tensor' has no attribute 'ham_heis'

I really appreciate any help.
Thanks

**Comment by jcmgray** [comment]
Ah yes there is a missing underscore it should be ``ham_1d_heis`` - https://quimb.readthedocs.io/en/latest/_autosummary/quimb.tensor.tensor_gen.html?highlight=ham_1d_heis#quimb.tensor.tensor_gen.ham_1d_heis. Docs need to be updated.

**Comment by saraphys** [comment]
Thanks for the reply.
I changed it to ham_1d_heis. Still getting the same error

**Comment by jcmgray** [comment]
Are you on the most recent version from github?

**Comment by saraphys** [comment]
yes

**Comment by saraphys** [comment]
Actually, there is no function called "ham_1d_heis" in tensor_gen.py produced in my quimb folder.

**Comment by saraphys** [comment]
This is my quimb version:
(cotengra) (base) user@amst4:~$ pip show quimb
Name: quimb
Version: 1.3.0+368.g9c26756
Summary: Quantum information and many-body library.
Home-page: http://quimb.readthedocs.io
Author: Johnnie Gray
Author-email: johnniemcgray@gmail.com
License: Apache
Location: /home/user/cotengra/lib/python3.8/site-packages
Requires: cytoolz, numba, numpy, psutil, scipy, tqdm
Required-by:

**Comment by jcmgray** [comment]
I can't recreate with a fresh install from github: https://colab.research.google.com/drive/1UAcO8pQUIU_kuHDuXXImSSojewRDL-Td?usp=sharing. 

My hunch would be that the installation has somehow broken (can happen quite easily) - try uninstalling including manually deleting the quimb folder from your `site-packages` and re-installing.

**Comment by saraphys** [comment]
Ok. Thank you. I will do that and update this comment.
Thank again

**Comment by saraphys** [comment]
Thank you the problem solved after reinstalling quimb.

**Comment by HamidArianZad** [comment]
Hi,

I have installed quimb on my system via Anaconda 3 (python-3.7).
All quimb dependencies have been installed, as well.
I am trying to test quimb to draw a tensor tree by running the defined code:

-----------------------------------------------------
from quimb import *
import quimb.tensor as qtn

Lx = Ly = Lz = 4
D = 2
tn = qtn.TN3D_rand(Lx, Ly, Lz, D=D)
tn.add_tag('CUBE')

color = ['CUBE'] + [
    f'I{i},{j},{k}'
    for i in (0, Lx - 1)
    for j in (0, Ly - 1)
    for k in (0, Lz - 1)
]

tn.draw(color=color)
-------------------------------------------------
but I get the error:

AttributeError: module 'quimb.tensor' has no attribute 'TN3D_rand'

Could you please help me to resolve this problem?

By the way, whenever I add line:
%config InlineBackend.figure_formats = ['svg']
my python does not recognize it.

**Comment by jcmgray** [comment]
Hi @HamidArianZad, please consider opening a separate issue if the fix in this one (i.e. a fresh install from github) does not work for you.

> By the way, whenever I add line:
> %config InlineBackend.figure_formats = ['svg']

This is just for jupyter notebooks (the format the docs are largely written in).
